### PR TITLE
feat(components): [check-tag] add type prop

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/check-tag/src/check-tag.vue [hunk 1]
```diff
@@ -17,7 +17,11 @@ const props = defineProps(checkTagProps)
 const emit = defineEmits(checkTagEmits)
 
 const ns = useNamespace('check-tag')
-const containerKls = computed(() => [ns.b(), ns.is('checked', props.checked)])
+const containerKls = computed(() => [
+  ns.b(),
+  ns.is('checked', props.checked),
+  ns.m(props.type || 'primary'),
```
Comment (btea) [inline packages/components/check-tag/src/check-tag.vue:23]: `props.type` has a default value, maybe we don't need to set the default value repeatedly?

Comment (kooriookami) [reply packages/components/check-tag/src/check-tag.vue:23]: It's for setting empty string to `type`.

Comment (btea) [reply packages/components/check-tag/src/check-tag.vue:23]: `type` does not seem to allow accepting empty strings.

Comment (kooriookami) [reply packages/components/check-tag/src/check-tag.vue:23]: Yes, It's an insurance operation. If someone set `null`, we need to use the default value.

**Review by btea — APPROVED** [review]

**Review by btea — APPROVED** [review]

**Pull request merged**